bundles/org.openhab.binding.daikin/src/main/java/org/openhab/binding/daikin/internal/api/ControlInfo.java

```diff
@@ -87,4 +105,21 @@ public Map<String, String> getParamString() {
     public SpecialMode getSpecialMode() {
         return SpecialMode.fromAdvancedMode(advancedMode);
     }
+
+    /*
+     * This method combines different direction parameters to one.
+     * 
+     * @param ud up/ down value S or 0
+     * 
+     * @param lr left/ right value S or 0
+     * 
+     * @return combined int value based on {@link FanMovement} enum
+     */
+    public static int parseFanMovement(String ud, String lr) {
+        if ("S".equals(ud)) {
+            return "S".equals(lr) ? FanMovement.VERTICAL_AND_HORIZONTAL.getValue() : FanMovement.VERTICAL.getValue();
+        } else {
+            return "S".equals(lr) ? FanMovement.HORIZONTAL.getValue() : FanMovement.STOPPED.getValue();
+        }
+    }
```

Comment:
Why not return the actual enum here, to avoid going back and forth with the enum -> value -> enum.

Reply:
Previously the method and call where in seperate files, that looked good. But adfter the movemt i forgot to adapt.